doc/Protocols/pr-meta_x-road_service_metadata_protocol.md

```diff
@@ -157,38 +161,58 @@ Annexes [C.4](#c4-listmethods-response) and [C.6](#c6-allowedmethods-response) c
         </xs:sequence>
     </xs:complexType>
 ```
-    
-## 5 Retrieving WSDL of a Service   
 
-Service clients are able to download WSDL files that contain the definition of a given service. 
-This can be accomplished by making HTTP GET request to the client's security server. 
-The URL of the request is either `http://SECURITYSERVER/wsdl` or `https://SECURITYSERVER/wsdl` depending on whether the HTTPS protocol is configured for interaction between the security server and the information system. 
-When making the request, the address `SECURITYSERVER` must be replaced with the actual address of the security server. The client MUST specify the identifier of the service using the following parameters:
+## 5 Retrieving the WSDL of a Service   
+
+Service clients are able to download WSDL-files that contain the definition of a given service by using the `getWsdl` meta-service. This can be accomplished by either sending the client security server an X-Road protocol POST-request or a parametrized HTTP GET-request.
+
+#### X-Road protocol POST-request
+
+  * the standard method for retrieving the WSDL
+  * uses the connection type settings of the client subsystem
+  * WSDL is retrieved as a SOAP-attachment
+
+An example of a `getWsdl` X-Road protocol POST-request to the client security server is documented in annex [C.7](#c7-getwsdl-request) and the corresponding response in annexes [C.8](#c8-getwsdl-response) and [C.9](#c9-getwsdl-response-attachment).
+
+#### HTTP GET-request
+
+  * a convenience method for retrieving the WSDL that will be phased out in future releases
+  * disabled by default in new `6.17.x` installations, availability is configured by the `allow-get-wsdl-request` system parameter \[[UG-SYSPAR](#Ref_UG-SYSPAR)\]
+  * uses the connection type settings of the owner member on the client security server
+  * WSDL is retrieved in the response body.
+
+The URL for the HTTP GET-request is either `http://SECURITYSERVER/wsdl` or `https://SECURITYSERVER/wsdl` depending on the connection type settings for the client owner member. When making the request, the address `SECURITYSERVER` must be replaced with the actual address of the client security server. The client MUST specify the identifier of the service using the following HTTP-parameters:
 
 * `xRoadInstance` – code that identifies the X-Road instance;
 
 * `memberClass` – code that identifies the member class;
 
 * `memberCode` – code that identifies the X-Road member;
 
-* `subsystemCode` (optional) – code that identifies a subsystem of the given member (if the service is provided by a subsystem);
+* `subsystemCode` – code that identifies a subsystem of the given member;
```

Comment:
We disabled the addition of services to members, but on protocol level (PR-MESS) it is still allowed. Also when loading WSDL of central service (defined in central server) it is enough to provide "xRoadInstance" and "serviceCode".